repos/system_upgrade/common/files/upgrade_paths.json

```diff
@@ -14,6 +15,7 @@
             "8.10": ["9.6", "9.4"],
             "8": ["9.6", "9.4"],
             "9.6": ["10.0"],
+            "9.7": ["10.1"],
```

Comment:
@karolinku that's incorrect. saphana is only and only for even releases. so saphnana should not be modified anyhow at this moment.